google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1alpha2/StreamWriter.java

```diff
@@ -800,6 +837,13 @@ public void onResponse(AppendRowsResponse response) {
         if (response == null) {
           inflightBatch.onFailure(new IllegalStateException("Response is null"));
         }
+        if (response.hasUpdatedSchema()) {
+          if (streamWriter.getOnSchemaUpdateRunnable() != null) {
```

Comment:
Is it allowed to have OnSchemaUpdateRunnable here? Or we should throw exception?

Reply:
Users can not set onSchemaUpdateRunnable and basically not handle what to do for schema updates.